### PR TITLE
store multiqc intermediate json files for rnafusion

### PR DESCRIPTION
**Pull request by fevac**
### Added

- Delivery of multiqc inftermediate files for rnafusion

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell

  bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b rnafusion_multiqc_json
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b deliver_multiqc_files -a

    ```

### How to test

- [x] `cg workflow rnafusion report-deliver tendergoose`
- [x] `cg workflow rnafusion store-housekeeper tendergoose`. Then check housekeeper and make sure intermediate qc files are added with the corresponding tags
 
image

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by henrikstranneheim — APPROVED** [review]
Nice, 👍 !

File: cg/resources/rnafusion_bundle_filenames.csv [hunk 1]
```diff
@@ -100,4 +100,44 @@
...
+  path: PATHTOCASE/multiqc/multiqc_data/multiqc_general_stats.json
+  path_index: ~
+  step: multiqc-json
+  tag: multiqc-general-stats
+- format: json
+  id: CASEID
+  path: PATHTOCASE/multiqc/multiqc_data/multiqc_picard_dups.json
+  path_index: ~
+  step: multiqc-json
+  tag: multiqc-picard-duplicates
```
Comment (ChrOertlin) [inline cg/resources/rnafusion_bundle_filenames.csv:126]: @ivadym can you have a look that we use similar tags as what has been implemented in balsamic?

**Review by ivadym — APPROVED** [review]
Nice ⭐

File: cg/resources/rnafusion_bundle_filenames.csv [hunk 2]
```diff
@@ -100,4 +100,38 @@
...
+  id: CASEID
+  path: PATHTOCASE/multiqc/multiqc_data/multiqc_star.json
+  path_index: ~
+  step: multiqc
+  tag: multiqc-star
+- format: json
+  id: CASEID
+  path: PATHTOCASE/multiqc/multiqc_data/multiqc_picard_insertSize.json
+  step: multiqc
+  tag: multiqc-picard-insert-size
```
Comment (ivadym) [inline cg/resources/rnafusion_bundle_filenames.csv:137]: EOF
```suggestion
  tag: multiqc-picard-insert-size
  
```

**Comment by fevac** [comment]
- Deployed

image

image

**Pull request merged**